### ISSUE TITLE
Modifications needed for multi-label classification

### ISSUE THREAD

**Issue by shan18** [XifengGuo/CapsNet-Keras#88]
If I want to apply this model to a multi-class multi-label classification, what changes would be required for me to make on the `Mask()` layer? Can you give me any hint?
Will there be any other changes that I would have to make?

Thank You.

**Comment by Ryanglambert** [comment]
If you read/run the code as is, you will see this is already a multiclass
classifier.

**Comment by XifengGuo** [comment]
@shan18 
By default, during training, the mask is provided by `y`, so you needn't change the line#56.   
If the accuracy does not improve, you can track the training loss to see a NaN?

**Comment by shan18** [comment]
Okay, understood. Thank you for your help.
